### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by Poilichon** [Syncplay/syncplay#473] [Archive]
Hello, as I said in comment I would like to work on the french traduction if you're OK. I started up to line 58 and I was thinking of the possibility that you would not be OK with that.

**Comment by Et0h** [comment]
Hi, thanks for getting in touch. There is currently a pull request for a French translation at #455 by @sosie-js  but it is waiting to be brought into line with the other languages before it is accepted. As such, it might be worth that you double check that translation in the first instance.

**Reply by Poilichon** [reply]
Thank you for your answer, I'm new here so I didn't see everything. Thank you for this wonderful project by the way, it realy answer a need for me
